internal/go.mod

```diff
@@ -0,0 +1,16 @@
+module github.com/open-telemetry/opentelemetry-collector-contrib/internal
+
+go 1.14
+
+require (
```

Comment:
I think this might have broken the tools version pinning.

On master if you `go install github.com/golangci/golangci-lint/cmd/golangci-lint` you get v1.24 but now you get v1.26. I tried doing it inside the internal directory but same thing happened. Not sure how to fix it.